Assets/MixedRealityToolkit-SDK/StandardAssets/Controllers/ViveControllers/Models/vr_controller_vive_1_5.obj.meta

```diff
@@ -0,0 +1,94 @@
+fileFormatVersion: 2
```

Comment:
Where did this (and all the other) controller model come from? Do we have the right license to redistribute them? I don't see any license files in this PR.